.gitignore

```diff
@@ -0,0 +1,11 @@
+# Editors and IDEs
```

Comment:
Editor and IDE stuff should go into your personal, user-wide gitignore. Otherwise projects would have to add gitignores for each and every editor in use

Reply:
fixed